### DISCUSSION TITLE
Check Offset before ripping

### DISCUSSION THREAD

**Discussion by Anthropos888** [enzo1982/freac#339] [Your Ideas]
Hi,
I had the following problem: every time after ripping a CD I got the message, that the AccurateRip checksums are different.
This issue could'nt be solved by choosing a different preset with better mp3 compression etc.
But it could be solved by going to the settings -> Components -> Verification -> AccurateRip verification -> Drives -> determine offset.

So I would strongly recommend that fre:ac determines the offset of the CD drive automatically before ripping a CD.

**Comment by Lee-Carre** [comment]
While yes, ideally, if automateable. However, AccurateRip calibration requires using a disc (in your drive) which already exists in AccurateRip's DB. There enlies the catch; that part isn't automatable. The user must find an already-known disc, to use.

This can be addressed with good UI design, and documentation (especially a section for newbies who want to set things up properly, rather than ‘quick start’).

Feel free to contribute documentation patches (via pull requests).
